packages/pg-pool/index.js

```diff
@@ -116,11 +116,20 @@ class Pool extends EventEmitter {
     const pendingItem = this._pendingQueue.shift()
     if (this._idle.length) {
       const idleItem = this._idle.pop()
-      clearTimeout(idleItem.timeoutId)
       const client = idleItem.client
-      const idleListener = idleItem.idleListener
+      clearTimeout(idleItem.timeoutId)
```

Comment:
Just curious: was there any particular reason why this line needed to be moved?

Reply:
not really, no

Reply:
np -- just wanted to confirm that there wasn't some important reason that I completely overlooked.  😅

Reply:
haha yeah nah just moving code slightly & bumped that line!